README.md

```diff
@@ -46,7 +46,7 @@ Cruise Control for Apache Kafka
     * Fix offline replicas
     * Adjust replication factor
 
-### Environment Requirements
+### Environment Requirements ###
```

Comment:
Other than their cosmetic use, the only other use of closing hashes that I know is to escape `#` sign in the header. Given that all the other headers have the closing hashes, this change is good for consistency -- thanks for this change!